--- xcache-3.0.0/xcache.ini~	2012-10-18 10:49:34.000000000 +0300
+++ xcache-3.0.0/xcache.ini	2012-11-21 17:37:53.122003769 +0200
@@ -1,9 +1,5 @@
-;; this is an example, it won't work unless properly configured into php.ini
 [xcache-common]
-;; non-Windows example:
-extension = xcache.so
-;; Windows example:
-; extension = php_xcache.dll
+extension = @extensiondir@/xcache.so
 
 [xcache.admin]
 xcache.admin.enable_auth = On
